### ISSUE TITLE
api.open-elevation.com is down!

### ISSUE THREAD

**Issue by OlivierB-OB** [Jorl17/open-elevation#54]
Hi,

First thanks a lot for your very handy service!
It is great!

The service open-elevation API is down at the moment!

Rgds,
Olivier

**Comment by EduardoNigro** [comment]
Thank you João.
And yes! I would agree with Olivier. It seems to be down.

**Comment by Jorl17** [comment]
Hi @OlivierB-OB and @EduardoNigro 

Thank you for your issue regarding this. Yesterday, when I saw @OlivierB-OB's message, I brought the system back up. It still seems to be up to me. 

Are you sure it's down for you @EduardoNigro ? What exactly do you mean by it being down?

**Comment by EduardoNigro** [comment]
I keep getting 504 errors on data that used to work a month ago...

**Comment by Jorl17** [comment]
Sounds like a caching issue (might be on our end). Can you give me a concrete request to try out?

**Comment by EduardoNigro** [comment]
It is working now! What the heck.
Muito obrigado. Esse API é do balacobaco!

**Comment by OlivierB-OB** [comment]
Hi @Jorl17 ,

From our end the service is actually back to normal!
Thanks a lot for tackling this quickly!

Here are a few feedbacks on what I have noticed on my end, just in case it could be useful.
On Monday we noticed that the service was down.
Actually after your first restart, Tuesday morning the service was back to normal.
But in the afternoon, it started to be unstable. Very unstable.
From time to time a call was successfully addressed, but most of the time the service was not responding.
It is now working fine...

Again, thanks a lot for your quick response, and for this awesome service! 😃 

Olivier
